src/index.html

```diff
@@ -6,13 +6,71 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Product cards</title>
+    <title>Product products</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__product-foto">
+        <img
+          class="card__photo"
+          src="images/imac.jpeg"
+          alt="product"
```

Comment:
alt attribute should describe the image content
<img width="456" alt="image" src="https://github.com/user-attachments/assets/1cbb2ca5-66cd-4586-b2ce-a5cec6422e1c">